.vscode/settings.json

```
@@ -189,5 +189,5 @@
   "evenBetterToml.formatter.reorderArrays": true,
   "evenBetterToml.formatter.trailingNewline": true,
   // We like keeping TOML keys in a certain non-alphabetical order that feels more natural
-  "evenBetterToml.formatter.reorderKeys": false
+  "evenBetterToml.formatter.reorderKeys": false,
```

Reducing changes in PR
```suggestion
  "evenBetterToml.formatter.reorderKeys": false
```